technical-reports/format/design-token.md

```diff
@@ -179,4 +179,35 @@ Tool vendors are encouraged to publicly share specifications of their extension
   The extensions section is not limited to vendors. All token users can add additional data in this section for their own purposes.
 </p>
 
-## More token properties TBC
+### Deprecated
+
+The optional **`$deprecated`** property is a boolean or string where tooks MAY specify deprecated tokens. A token MAY be marked deprecated in any of the following scenarios:
```

Comment:
cc @c1rrus any nitpicking of language here would be greatly appreciated 🙏. Happy to rephrase this to match the existing language (or if you wanted to do a pass on it that would be fine, too!)